index.js

```diff
@@ -1,5 +1,5 @@
 module.exports = [
-    "> 0.2%",
+    "last 1 year or > 0.2%",
```

Comment:
nit: This can be split in two lines since the `or` operator is used across lines.
```suggestion
    "last 1 year",
    "> 0.2%",
```

Reply:
Fixed in a new commit